openwisp_radius/api/urls.py

```diff
@@ -11,6 +11,16 @@
     url(r'^registration/(?P<slug>[\w-]+)/$',
         views.register,
         name='rest_register'),
+    # password reset
+    url(r'^password/reset/confirm/$',
```

Comment:
we need to implement the same functionality which checks the org here, remember: these URLs are used by captive pages that are specific to each organization, it's like they are totally separate instances.

In some other URLs of the API we don't have the `slug` parameter because the organization data is sent in the authorization header with a Bearer token, eg: `Authorization: Bearer {uuid} {token}`, all API URLs have a way to separate the different organizations using the system.